lib/login_page1.dart

```diff
@@ -0,0 +1,220 @@
+import 'package:flutter_riverpod/flutter_riverpod.dart';
```

Comment:
why do we have login_page1 instead of login_page now?

Reply:
When I first started I created a new file and didn't remove the login_page file yet. Now I removed the login_page file but I didn't change this file's name. Would it be better to change its name to 'login_page' for consistency? Then I will do that